src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -65,12 +74,22 @@ const CenterIconText = styled(IconText)`
   margin: 16px 55px 20px 0;
 `
 
+const AddCustomSafeAppCard = styled(Card)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 164px;
```

Comment:
Do we need a fixed height for this element if its already part of the grid?

Reply:
Its needed to make sure that the "Add Custom App" Card has the same height than the other Safe App Cards. The edge case is when its the only card in the row (very small resolutions):
![Captura de pantalla 2022-04-26 a las 18 11 48](https://user-images.githubusercontent.com/26763673/165345246-bc21d518-cfc1-4dcb-aaaf-e9729552b7ec.png)